beps/bal-graphql/1493_Revamping Ballerina GraphQL CLI Tool.md

```
@@ -0,0 +1,787 @@
+# **\#1493 :** **Revamping Ballerina GraphQL CLI Tool**
+
+- Authors: Nidula Ekanayake
+- Reviewed by: Thisaru Guruge, Danesh Kuruppu
+- Created date: 01/08/2026
+- Issue: [#1493](https://github.com/ballerina-platform/ballerina-spec/issues/1493)
+- State: Submitted
+
+### **1\. Summary**
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)